src/views/Change.vue

```diff
@@ -44,26 +44,17 @@
           @valid="setDocumentOptionalEmailValidity($event)"
         />
 
-        <section id="completing-party-section">
-          <h2>2. Completing Party</h2>
-          <v-card flat>
```

Comment:
Most of this is moved to "shim" component, so that THIS page is clean and consistent.